.github/workflows/rust.yml

```diff
@@ -208,7 +208,7 @@ jobs:
 
       - name: Run lints
         working-directory: ${{ matrix.directory }}
-        run: cargo +${{ matrix.toolchain }} make --profile ${{ matrix.profile }} clippy -- -D warnings
+        run: cargo +${{ matrix.toolchain }} make --profile ${{ matrix.profile }} lint
```

Comment:
Is this equivalent?

Reply:
Lint will do `rustfmt`, `clippy` and `doc`, so it's a superset. `cargo make doc` is not tested (and failed), only `cargo make rustdoc` is tested.